CHEWBBACA/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "2.1.0"
+__version__ = "2.5.0"
```

Comment:
from 2.1.0 to 2.5.0? I don't understand the large jump

Reply:
We're introducing a lot of changes. These changes are not enough to increment the major version but are significant and jumping several minor versions emphasizes that.